micro-demos/services/demo.sh

```diff
@@ -15,18 +15,18 @@
 
 . $(dirname ${BASH_SOURCE})/../util.sh
 
-if kubectl --namespace=demos get rc hostnames >/dev/null 2>&1; then
-    desc "Revisit our replication controller"
-    run "kubectl --namespace=demos get rc hostnames"
+if kubectl --namespace=demos get rs hostnames >/dev/null 2>&1; then
+    desc "Revisit our replica set"
+    run "kubectl --namespace=demos get rs hostnames"
 else
-    desc "Run some pods under a replication controller"
-    run "kubectl --namespace=demos run hostnames \\
```

Comment:
I really want this demo to use run and expose.  You can switch it to not re-use the old rs, and change it to poke at deployments rather than RCs